keel-sql/keel-sql.gradle.kts

```diff
@@ -37,7 +37,10 @@ dependencies {
   testImplementation("io.strikt:strikt-core")
   testImplementation(project(":keel-spring-test-support"))
   testImplementation(project(":keel-core-test"))
-  testImplementation(project(":keel-api"))
+  testImplementation(project(":keel-api")) {
+    // avoid circular dependency which breaks Liquibase
+    exclude(module = "keel-sql")
```

Comment:
this upcoming gradle feature might address this: https://docs.gradle.org/5.6-rc-1/release-notes.html#test-fixtures-for-java-projects (but this makes sense for me for now)